File: java-main/java-main/JavaDSA_Super-X/Assi_4/p2.java
import java.util.Scanner;

class Demo {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the number of rows: ");
        int numRows = scanner.nextInt();
        for (int row = 1; row <= numRows; row++) {
            for (char ch = (char)('A' + row - 1); ch >= 'A'; ch--) {
                System.out.print(ch + " ");
            }
            System.out.println(); 
        }

        scanner.close();
    }
}

File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/session6/p2.c
//1  2  3  4 
//   5  6  7
//      8  9
//         10

#include<stdio.h>
void main(){
	int row;
	printf("enter the row:\n");
	scanf("%d",&row);

	int num=1;
	for(int i=1;i<=row;i++){
		for(int sp=1;sp<i;sp++){
			printf(" \t");
		}
		for(int j=row;j>=i;j--){
			printf("%d\t",num);
			num++;
		}
		printf("\n");

	}
}
